src/components/common/hairlineSeparator.tsx

```diff
@@ -25,7 +25,7 @@ export function HairlineSeparator({
       style={[
         styles.HairlineWidth,
         {
-          borderColor: colors.primary.separtorColor,
+          borderColor: colors.primary.separatorColor,
```

Comment:
Thanks for fixing my spelling mistakes 😬

Reply:
haha, no worries. my editor picked it up so I realized might as well fix this too